public/application.js

```diff
@@ -10,4 +10,14 @@ $(document).ready(function(){
 	$('#findRecipes').on('click', function(){
 		window.location.href="findRecipe"
 	})
+
+	$('#logout').on('click', function(){
```

Comment:
Isn't it duplicated with https://github.com/WDI-HK-8/Recipz/pull/2/files#diff-d2828a7892ad0ab128616412d743881fR88 ?